### ISSUE TITLE
Need internal logging mechanism

### ISSUE THREAD

**Issue by jnooree** [usr-sse2/RDM#20]
This utility relies on private interface and directly accesses low-level graphics. Since there is no guarantee that those low-level jobs work properly and consistently, there should be some kind of logging mechanism for this app. It would make debugging much easier, and finding out the source of the problem.

[This doc](https://developer.apple.com/documentation/os/logging) might be helpful. I'll try to work on this stuff, but after #19 is merged into. I think it is not a good idea to add more commits to the PR that is already under reviewing process.
